I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 1725, in script
    $ renpy.pause(hard=True)
  File "game/script.rpy", line 1725, in <module>
    $ renpy.pause(hard=True)
  File "game/scripts/HUD.rpy", line 159, in execute
    screen journal_screen():
  File "game/scripts/HUD.rpy", line 159, in execute
    screen journal_screen():
  File "game/scripts/HUD.rpy", line 165, in execute
    frame:
  File "game/scripts/HUD.rpy", line 171, in execute
    vbox:
  File "game/scripts/HUD.rpy", line 190, in execute
    if current_tab == "Missions":
  File "game/scripts/HUD.rpy", line 345, in execute
    viewport:
  File "game/scripts/HUD.rpy", line 351, in execute
    vbox:
  File "game/scripts/HUD.rpy", line 356, in execute
    if len(collected_tapes) > 0:
  File "game/scripts/HUD.rpy", line 356, in <module>
    if len(collected_tapes) > 0:
NameError: name 'collected_tapes' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 1725, in script
    $ renpy.pause(hard=True)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\ast.py", line 823, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 1725, in <module>
    $ renpy.pause(hard=True)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\exports.py", line 1661, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward, pause=delay, pause_modal=modal)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\core.py", line 2165, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\core.py", line 2687, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\displayable.py", line 431, in visit_all
    d.visit_all(callback, seen)
  [Previous line repeated 1 more time]
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\screen.py", line 480, in visit_all
    callback(self)
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\core.py", line 2687, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\screen.py", line 491, in per_interact
    self.update()
  File "D:\renpy\renpy-8.0.1-sdk\renpy\display\screen.py", line 697, in update
    self.screen.function(**self.scope)
  File "game/scripts/HUD.rpy", line 159, in execute
    screen journal_screen():
  File "game/scripts/HUD.rpy", line 159, in execute
    screen journal_screen():
  File "game/scripts/HUD.rpy", line 165, in execute
    frame:
  File "game/scripts/HUD.rpy", line 171, in execute
    vbox:
  File "game/scripts/HUD.rpy", line 190, in execute
    if current_tab == "Missions":
  File "game/scripts/HUD.rpy", line 345, in execute
    viewport:
  File "game/scripts/HUD.rpy", line 351, in execute
    vbox:
  File "game/scripts/HUD.rpy", line 356, in execute
    if len(collected_tapes) > 0:
  File "game/scripts/HUD.rpy", line 356, in <module>
    if len(collected_tapes) > 0:
NameError: name 'collected_tapes' is not defined

Windows-10-10.0.26100 AMD64
Ren'Py 8.2.1.24030407
The Merge 1.0
Mon Apr 14 20:31:06 2025
